va/va.go

```diff
@@ -1092,7 +1093,7 @@ func (va *ValidationAuthorityImpl) PerformValidation(ctx context.Context, domain
 	logEvent.Challenge = challenge
 
 	validationLatency := time.Since(vStart)
-	logEvent.ValidationLatency = validationLatency
+	logEvent.ValidationLatency = math.Floor(1000*float64(validationLatency)/float64(time.Second)) / 1000
```

Comment:
This and below ends up being a bit of a pain to read, can we just use the `time` builtins for this conversion? i.e. `validationLatency.Round(time.Millisecond).Seconds()`.